src/api/goods.ts

```diff
@@ -1,11 +1,18 @@
 // eslint-disable-next-line
 const API_URL = `https://mate-academy.github.io/react_dynamic-list-of-goods/goods.json`;
 
-export function getAll(): Promise<Good[]> {
-  return fetch(API_URL)
-    .then(response => response.json());
+export async function getAll(): Promise<Good[]> {
+  return (await fetch(API_URL)).json();
```

Comment:
yeah, this syntax is shorter, but it's hard to read, MB better to save the fetch result to a variable?